README.md

```diff
@@ -1,2 +1,103 @@
-# module-ballerinai-observe
-Observability components related to the internal executions in Ballerina
+
+Ballerina Observe Internal Library
+===================
+
+[![Build](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/build-timestamped-master.yml)
+[![Trivy](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/trivy-scan.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinai-observe.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/commits/master)
+[![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/http.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fhttp)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinai-observe/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinai-observe)
+
+This module contains internal configurations and initializations for Ballerina Observability. Ballerina supports Observability out of the box. You can use [module-ballerina-observe](https://github.com/ballerina-platform/module-ballerina-observe) in your ballerina project and enable observability features.
+
+By default, observability is not included in the executable created by Ballerina. It can be added by using the `--observability-included` build flag or by adding the following section to the `Ballerina.toml` file.
+
+
+
+## Issues and projects
+
+Issues and Projects tabs are disabled for this repository as this is part of the Ballerina Standard Library. To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```

Comment:
@anupama-pathirage @shafreenAnfar Should we enable issues & projects for this repo or consider these as part of the standard library ? 🤔

I'm asking because we were previously requested not to consider this as a standard library.